lbrynet/core/StreamDescriptor.py

```diff
@@ -445,7 +445,10 @@ def download_sd_blob(session, blob_hash, payment_rate_manager, timeout=None):
                                           payment_rate_manager,
                                           session.wallet,
                                           timeout)
+    mirror = HTTPBlobDownloader(session.blob_manager, [blob_hash], session.download_mirrors)
+    mirror.start()
```

Comment:
Does this need to be `yield`ed?

Reply:
No, the idea is that it doesn't care about the result of `start` at all. Yielding would also make it wait, but we agreed to leave it concurrently.